enterprise/fulltext-addon/src/test/java/org/neo4j/kernel/api/impl/fulltext/integrations/bloom/BloomIT.java

```diff
@@ -52,6 +56,8 @@
     public final FileSystemRule fs = new DefaultFileSystemRule();
```

Comment:
Using DefaultFileSystemRule` as type avoids raw usage. The static fields could be private as well.